### PR TITLE
add default params for the Integer field type on install

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue #15394 .

### Summary of Changes
make sure that the integer field type is usable

### Testing Instructions
BEFORE PR
On a Clean install 
Create a content field of type integer. DO not set any values in the First, Last or Step fields
Try to use the field in an article

### Expected result
The field has the first, last and step values from the plugin of 1, 100, 1

### Actual result
The field can not be used

### This PR
This PR insures that the default values of 1,100,1 are set for the params in the database on install.

### Testing
This cannot be tested with com_patchtester as it just modifies the sql file that is run on install.

You can test it by manually inserting 
`{"multiple":"0","first":"1","last":"100","step":"1"}` in the params field for plg_fields_integer in the #__extensions table

### CODE REVIEW

**Comment by brianteeman** [comment]
@rdeutz @zero-24 This really needs to be merged into 3.7.0 or the integer field is pretty useless

**Comment by rdeutz** [comment]
At least one test would good before we merge it

**Comment by brianteeman** [comment]
@rdeutz absolutely I wasnt suggesting otherwise - just that it needs to be merged in 3.7.0 when it is tested

**Comment by franz-wohlkoenig** [comment]
will test in a couple of Minutes.

File: installation/sql/sqlazure/joomla.sql [hunk 1]
```diff
@@ -851,7 +851,7 @@ INSERT INTO "#__extensions" ("extension_id", "package_id", "name", "type", "elem
 (464, 0, 'plg_fields_color', 'plugin', 'color', 'fields', 0, 1, 1, 0, '', '', '', '', 0, '1900-01-01 00:00:00', 0, 0),
 (465, 0, 'plg_fields_editor', 'plugin', 'editor', 'fields', 0, 1, 1, 0, '', '', '', '', 0, '1900-01-01 00:00:00', 0, 0),
 (466, 0, 'plg_fields_imagelist', 'plugin', 'imagelist', 'fields', 0, 1, 1, 0, '', '', '', '', 0, '1900-01-01 00:00:00', 0, 0),
-(467, 0, 'plg_fields_integer', 'plugin', 'integer', 'fields', 0, 1, 1, 0, '', '', '', '', 0, '1900-01-01 00:00:00', 0, 0),
+(467, 0, 'plg_fields_integer', 'plugin', 'integer', 'fields', 0, 1, 1, 0, '', '{"multiple":"0","first":"1","last":"100","step":"1"}, '', '', 0, '1900-01-01 00:00:00', 0, 0),
```
Comment (zero-24) [inline installation/sql/sqlazure/joomla.sql]: the closing `'` is missing

**Comment by franz-wohlkoenig** [comment]
@brianteeman 

- applied `{"multiple":"0","first":"1","last":"100","step":"1"}` in params of plg_fields_integer
- new Field "Integer" > Dropdown isn't clickable.

**Comment by brianteeman** [comment]
in the extensions table?

**Comment by franz-wohlkoenig** [comment]
sorry, wrong Table. In "_extensions" i can't find a `plg_fields_integer`-Field and dont know how to append it.

**Comment by brianteeman** [comment]
If you dont have a record for the plugin then you have done something very wrong with your install

screenshotr11-13-56

**Comment by zero-24** [comment]
Don't we also need to add that to the initial update sql file?

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on c0c87cd1a0db59698c353ecf2145b077f5ba423e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15403">issues.joomla.org/tracker/joomla-cms/15403</a>.</sub>

**Comment by brianteeman** [comment]
@zero-24 why do we need it in the update - there is no update path from a beta or RC release and these fields never existed before

**Comment by Bakual** [comment]
@brianteeman Tobias means here: https://github.com/joomla/joomla-cms/blob/staging/administrator/components/com_admin/sql/updates/mysql/3.7.0-2016-08-29.sql#L84
and in the respective other two files.
That is needed for updates, the one you did is for new installs which is of course needed as well.

**Comment by brianteeman** [comment]
@bakual @zero-24 ah - i understand now. Do I just edit that file?

**Comment by brianteeman** [comment]
got it - update coming

**Comment by Bakual** [comment]
Yes, that will work since we don't support (yet) updating from RCs.

**Comment by brianteeman** [comment]
@zero-24 update scripts done

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 1c6f46f223af06345747835ab98ddccbde068f88<br /><br />Tested on MySQL only.
Did a clean install and worked as expected.
Did the upgrade and manually ran the update sql in phpMyAdmin.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15403">issues.joomla.org/tracker/joomla-cms/15403</a>.</sub>

**Comment by brianteeman** [comment]
thanks

**Pull request merged**